schema/deploy/shipping/views.sql

```diff
@@ -2089,7 +2089,10 @@ create or replace view shipping.scan_return_results_v1 as
       select
         sample_id,
         barcode as qrcode,
-        encountered::date as collect_ts,
+        case when sample.created >= '2020-08-14'
```

Comment:
Can we use another date field as the cutoff? I think it should be a date that we can easily point to for people outside the dev team. `sample.created` doesn't necessarily correlate with any specific date outside the database.

Reply:
Hmm. We do expose it via the [Sample Lookup](https://backoffice.seattleflu.org/metabase/dashboard/45) dashboard in Metabase, but still, a good point. I used `sample.created` because it's more stable and reliable than other dates so the condition wouldn't shift over time. Do you have any suggestions on a better date field to use?

Reply:
Posting here for posterity's sake:
I was thinking encountered date would be best? That way we can say any participant who enrolled after X date will have the corrected collection date.

Reply:
(And that's what we decided to do. :-)